website/docs/docs/core/about-core-setup.md

```diff
@@ -14,7 +14,9 @@ dbt Core is an [open-source](https://github.com/dbt-labs/dbt-core) tool that ena
 - [Connecting to a data platform](/docs/core/connect-data-platform/profiles.yml)
 - [How to run your dbt projects](/docs/running-a-dbt-project/run-your-dbt-projects)
 
-To learn about developing dbt projects in dbt Cloud, refer to [Develop with dbt Cloud](/docs/cloud/about-develop-dbt).
-  - dbt Cloud provides a command line interface with the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation). Both dbt Core and the dbt Cloud CLI are command line tools that let you run dbt commands. The key distinction is the dbt Cloud CLI is tailored for dbt Cloud's infrastructure and integrates with all its [features](/docs/cloud/about-cloud/dbt-cloud-features).
+You can also learn about [developing dbt projects in dbt Cloud](/docs/cloud/about-develop-dbt) &mdash; dbt Cloud provides a command line interface with the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) that let you run dbt commands. The dbt Cloud CLI is tailored for dbt Cloud's infrastructure and integrates with all its [features](/docs/cloud/about-cloud/dbt-cloud-features).
```

Comment:
This section feels really out of place and breaks up the flow of this page. I kind of understand why we have it (push people to Cloud) but there could be a better way to approach this without plopping a paragraph of text at the beginning of a Core instructional page. 

We can at least clarify the language so it's less jarring
```suggestion
If you're interested in using a command line interface to [develop dbt projects in dbt Cloud](/docs/cloud/about-develop-dbt), the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) lets you run dbt commands locally. The dbt Cloud CLI is tailored for dbt Cloud's infrastructure and integrates with all its [features](/docs/cloud/about-cloud/dbt-cloud-features).
```